libr/core/cmd.c

```diff
@@ -5172,7 +5165,27 @@ DEFINE_HANDLE_TS_FCN(help_command) {
 		recursive_help (state->core, detail, node_string);
 		return R_CORE_CMD_STATUS_OK;
 	} else {
-		return int2cmdstatus(r_cmd_call (state->core->rcmd, node_string));
+		TSNode command = ts_node_child_by_field_name (node, "command", strlen ("command"));
+		char *command_str = ts_node_sub_string (command, state->input);
+		TSNode args = ts_node_child_by_field_name (node, "args", strlen ("args"));
+		RCmdParsedArgs *pr_args = NULL;
+		int res = -1;
+		if (!ts_node_is_null (args)) {
+			pr_args = ts_node_handle_arg_prargs (state, node, args, 1);
+			if (!pr_args) {
+				goto err_else;
+			}
+			r_cmd_parsed_args_setcmd (pr_args, command_str);
+		} else {
+			pr_args = r_cmd_parsed_args_newcmd (command_str);
```

Comment:
Doesn't this need has_space_after_cmd handling?

Reply:
no, i'm handling this in exec_string so it does not print the space if there are no additional args apart from argv[0].

Reply:
Then what about the case with args and no space?

Reply:
That's handled at https://github.com/radareorg/radare2/pull/16703#discussion_r413905943

Reply:
How does handling it for completely different command "arged_command" help handling it for "help_command"?